1.8/usage/managing-services/list.md

```diff
@@ -21,4 +21,4 @@ marathon  <privatenode3>   True     3    2.0   1843.0  100.0  <service-id4>
 
 From the DC/OS web interface, click the [**Services**](/docs/1.8/usage/webinterface/#services) tab. 
 
-![Services](../img/dcos-services.png)
```

Comment:
this link really was broken, though, I believe. Or, at least, the image doesn't show up and the name was wrong I believe.